indico/modules/events/contributions/__init__.py

```diff
@@ -94,3 +94,20 @@ class SubmitterRole(ManagementRole):
     name = 'submit'
     friendly_name = _('Submission')
     description = _('Grants access to materials and minutes.')
+
+
+@signals.event.sidemenu.connect
+def _extend_event_menu(sender, **kwargs):
+    from indico.modules.events.contributions.util import get_contributions_with_user_as_submitter
+    from indico.modules.events.layout.util import MenuEntryData
+
+    def _visible_my_contributions(conf):
+        return session.user and bool(get_contributions_with_user_as_submitter(conf.as_event, session.user))
+
+    def _visible_list_of_contributions(conf):
+        return bool(conf.as_event.contributions)
```

Comment:
this triggers a lazyload on the whole contribution list of the event. i'd rather query whether there are any contributions for that event